packages/dashboards/examples/src/components/test/visual/timeseries/widgets.ts

```diff
@@ -746,7 +746,13 @@ export const widgetConfigs: IWidget[] = [
                             leftAxisLabel: "Utilization (%)",
                             preset: TimeseriesChartPreset.StatusBar,
                             collectionId: collectionId2,
-                        } as ITimeseriesWidgetConfig,
+                            allowLegendMenu: true,
+                            projectType:
+                                TimeseriesWidgetProjectType.PerfstackApp,
+                            gridConfig: {
+                                fixedLayout: true,
```

Comment:
please create a new widgets that will have PerfstackApp projectType instead of modifying existing.